### PR TITLE
Add bigram to vectorizer

### PR DESCRIPTION
**Pull request by ayush-1506**
Fixes : #638 

Also fixes the re-occurrence of bug in it's get_similar_bugs list.

### CODE REVIEW

**Comment by marco-c** [comment]
> Also fixes the re-occurrence of bug in it's get_similar_bugs list.

It's preferable to do logic changes in separate commits, but it's fine this time

File: bugbug/similarity.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def get_similar_bugs(self, query, k=10):
 
 class NeighborsSimilarity(BaseSimilarity):
     def __init__(self, k=10, vectorizer=TfidfVectorizer):
-        self.vectorizer = TfidfVectorizer()
+        self.vectorizer = vectorizer(ngram_range=(1, 2))
```
Comment (marco-c) [inline bugbug/similarity.py]: I would make this an option. The caller can use whatever vectorizer they want, with a default of TfidfVectorizer.
The vectorizer can be a parameter of the __init__ function.
Either:
```
    def __init__(self, k=10, vectorizer_class=TfidfVectorizer):
        self.vectorizer = vectorizer_class()
```
or:
```
    def __init__(self, k=10, vectorizer=TfidfVectorizer()):
        self.vectorizer = vectorizer
```

File: bugbug/similarity.py [hunk 2]
```diff
@@ -172,4 +172,4 @@ def get_similar_bugs(self, query):
         processed_query = self.vectorizer.transform([get_text(query)])
         _, indices = self.similarity_calculator.kneighbors(processed_query)
 
-        return [self.bug_ids[ind] for ind in indices[0]]
+        return [self.bug_ids[ind] for ind in indices[0]].remove(query["id"])
```
Comment (marco-c) [inline bugbug/similarity.py]: Nit: this is nicer and has the same time complexity:
```
return [self.bug_ids[ind] for ind in indices[0] if self.bug_ids[ind] != query["id"]]
```

File: bugbug/similarity.py (hunk 1 above)
Comment (ayush-1506) [reply bugbug/similarity.py]: Isn't the first option you defined above same as what I'm doing right now?

Comment (marco-c) [reply bugbug/similarity.py]: You are forcing the usage of ngram_range=(1, 2)

**Comment by marco-c** [comment]
Can you add a comment with the results of this, compared to 1-gram?

**Pull request merged**